modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -35,4 +59,37 @@
     public Loza(ClusterService clusterNetSvc) {
         this.clusterNetSvc = clusterNetSvc;
     }
+
+    /**
+     * Start Raft manager.
+     */
+    public void start() {
```

Comment:
Why we need explicit start? What about calling `this.raftServer = new RaftServerImpl(clusterNetSvc, FACTORY, 1000, Map.of());` in constructor?